models/maespa/inst/template.job

```diff
@@ -15,7 +15,8 @@ if [ ! -e "@OUTDIR@/.dat" ]; then
   cd "@RUNDIR@"
   cp -r  /fs/data5/pecan.models/maespa/inputfiles/example2 "@RUNDIR@"
   cd example2
-
+  rm met.dat
+  cp @SITE_MET@ .
```

Comment:
Is there a reason to copy the site met to the local folder rather than creating a symbolic link?